action.yml

```diff
@@ -17,7 +17,7 @@ inputs:
     description: 'Path to the SQL database project file to deploy'
     required: false
   arguments:
-    description: 'In case DACPAC option is selected, additional SqlPackage arguments that will be applied. When SQL query option is selected, additional sqlcmd arguments will be applied.'
```

Comment:
are there any customers using additional sqlcmd arguments now that might have their scenario broken by this change?

Reply:
That is a great point and to be honest, we don't know since we don't have any telemetry to support this. It's a trade off between AAD support or additional SQLCMD args, and in v2 we opted for the AAD support instead since we'll still have support for most script scenarios.

Reply:
FWIW it did break our own PR check that used SQLCMD variables to delete the correct test DB afterwards, which now I have to use an additional yml step to replace the DB name first